pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -146,11 +146,13 @@ func classifyNodes(npm NodePodsMap, thresholds api.ResourceThresholds, targetThr
 		} else {
 			glog.V(2).Infof("Node %#v is appropriately utilized with usage: %#v", node.Name, usage)
 		}
-		glog.V(2).Infof("allPods:%v, nonRemovablePods:%v, bePods:%v, bPods:%v, gPods:%v", len(allPods), len(nonRemovablePods), len(bePods), len(bPods), len(gPods))
 	}
 	return lowNodes, targetNodes
 }
 
+// evictPodsFromTargetNodes evicts pods based on priority, if all the pods on the node have priority, if not
+// evicts them based on QoS as fallback option.
```

Comment:
The above is not correct, it should be "pods are evicted in the order of their priorities from low to high. Pods with same priority are evicted based on their QoS." QoS is not a fallback option, both priority and QoS work together.

Reply:
So, this is when any of the pods on the node have no priority associated with them, if we have atleast one pod with no priority then we will evict all the pods based on QoS.

Reply:
> So, this is when any of the pods on the node have no priority associated with them, if we have atleast one pod with no priority then we will evict all the pods based on QoS.

That approach seems incorrect. We should not do that. Anyway I think, either all pods have priority or no pods have priority (mainly in older release). If priority admission plugin is enabled, there will not be any mixed scenario. 

Even if there is a chance of mixed case, we should consider all pods with out priority as with same priority and evict them based on their QoS.

Reply:
> out priority as with same priority

You mean update the priority to 0 or something else? If yes, I think we shouldn't touch priority field.

Reply:
> You mean update the priority to 0 or something else? If yes, I think we shouldn't touch priority field.

I did not mention about updating priority field at all. I just mentioned "treating/grouping them as with the same priority".

Reply:
Can you please elaborate. We are sorting based on priority field.